scripts/convert_icon.py

```
@@ -44,37 +64,39 @@ def image_to_c_array(img, array_name):
                     v = pixels[y * width + x + b]
                     # 1 for white, 0 for black
                     bit = 1 if v >= threshold else 0
-                    byte |= (bit << (7 - b))
+                    byte |= bit << (7 - b)
             packed.append(byte)
     # Format as C array
-    c = f'#pragma once\n#include <cstdint>\n\n'
-    c += f'// size: {width}x{height}\n'
-    c += f'static const uint8_t {array_name}[] = {{\n    '
+    c = f"#pragma once\n#include <cstdint>\n\n"
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)